lib/tasks/tests.rake

```diff
@@ -2,6 +2,18 @@
 
 namespace :tests do
   namespace :migrations do
+    desc 'Prepares all migrations and test data for consistency checks'
+    task prepare_database: :environment do
+      {
+        'v2' => 2017_10_10_025614,
+        'v2_4' => 2018_05_14_140000,
+        'v2_4_3' => 2018_07_07_154237,
+      }.each do |release, version|
+        system("./bin/rails db:migrate VERSION=#{version}")
```

Comment:
This `system` call is here because I was struggling to get multiple `Rake::Task...invoke` calls to the migrator to actually do all the migrations. I suspect that because it was all in one process it was caching the target version of the first run, and then doing nothing on subsequent runs, so it would eventually hit an error when the data populate step tried to put something in place on a table that should have been there but wasn't.

A better solution here would be to figure out how to unset that migration version cache (or whatever similar issue is happening).

Reply:
Found a different way to do this, I think.